emulator/iSkipper/examples/capture/capture.ino

```diff
@@ -3,13 +3,14 @@
 #include <string.h>
 
 /* UPDATE THESE FOR YOUR PARTICULAR BOARD */
-#define IS_RFM69HW false //make true if using w version
+#define IS_RFM69HW true //make true if using w version
```

Comment:
I think this is a great change. I think a reasonable default should just be out of the box support for the feather.